### DISCUSSION TITLE
Rename 'Read' button to match current state

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2735] [Ideas]
### Idea Description

On a series page, if you haven't started a series yet I think the button should say "Start Reading" or something to that effect. That way it's easier to tell that you haven't started a series. 

If your caught up with reading it (no unread left), it could switch to "Done" or something similar. 

### Idea Category

User Experience

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by Hobogrammer** [comment]
Instead of "Done" for a completed series maybe "Re-read" or "Read again"?

**Reply by DieselTech** [reply]
Oh the reason I didn't mention that is there is discussion around re-reading a series and making it count again. This isn't set in stone yet, but marking a series to re-read will increase it's read count by +1, then wipe any current progress and let you restart again. That would allow users to reread series multiple times and still track how many times they have done so.

**Comment by majora2007** [comment]
Going to close this out since when we brought functionality to re-read series, the names now align with suggestions.
